src/Metadata/Property/Factory/SerializerPropertyMetadataFactory.php

```diff
@@ -17,6 +17,7 @@
 use ApiPlatform\Metadata\Exception\ResourceClassNotFoundException;
 use ApiPlatform\Metadata\ResourceClassResolverInterface;
 use ApiPlatform\Metadata\Util\ResourceClassInfoTrait;
+use Doctrine\ORM\Mapping\DiscriminatorMap;
```

Comment:
not a huge fan of using doctrine in this file, it'd be best inside DoctrineOrmPropertyMetadataFactory?